tests/qos/test_qos_sai.py

```diff
@@ -1107,7 +1107,8 @@ def testQosSaiLossyQueue(
     @pytest.mark.parametrize("LossyVoq", ["lossy_queue_voq_1"])
     def testQosSaiLossyQueueVoq(
         self, LossyVoq, ptfhost, dutTestParams, dutConfig, dutQosConfig,
-            ingressLossyProfile, duthost, localhost, get_src_dst_asic_and_duts
+            ingressLossyProfile, duthost, localhost, get_src_dst_asic_and_duts,
+            separated_dscp_to_tc_map_on_uplink
```

Comment:
since "separated_dscp_to_tc_map_on_uplink" is not fixture, need to modify, and can reference below changes
```
 from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory                     # noqa F401
 from tests.common.fixtures.ptfhost_utils import copy_saitests_directory                     # noqa F401
 from tests.common.fixtures.ptfhost_utils import change_mac_addresses                        # noqa F401
+from tests.common.fixtures.duthost_utils import dut_qos_maps_module                         # noqa F401
 from tests.common.fixtures.ptfhost_utils import ptf_portmap_file                            # noqa F401
 from tests.common.dualtor.dual_tor_utils import dualtor_ports, is_tunnel_qos_remap_enabled  # noqa F401
-from tests.common.helpers.assertions import pytest_assert
+from tests.common.helpers.assertions import pytest_require, pytest_assert
 from tests.common.helpers.pfc_storm import PFCStorm
 from tests.pfcwd.files.pfcwd_helper import set_pfc_timers, start_wd_on_ports
 from .qos_sai_base import QosSaiBase
@@ -1108,7 +1109,7 @@ class TestQosSai(QosSaiBase):
     def testQosSaiLossyQueueVoq(
         self, LossyVoq, ptfhost, dutTestParams, dutConfig, dutQosConfig,
             ingressLossyProfile, duthost, localhost, get_src_dst_asic_and_duts,
-            skip_src_dst_different_asic,  separated_dscp_to_tc_map_on_uplink
+            dut_qos_maps_module
     ):
         """
             Test QoS SAI Lossy queue with non_default voq and default voq
@@ -1127,6 +1128,9 @@ class TestQosSai(QosSaiBase):
             Raises:
                 RunAnsibleModuleFail if ptf test fails
         """
+        pytest_require(separated_dscp_to_tc_map_on_uplink(dut_qos_maps_module),
+                    "Skip test because separated QoS map is not applied")
+
         if dutTestParams["basicParams"]["sonic_asic_type"] != "cisco-8000":
             pytest.skip("Lossy Queue Voq test is not supported")
         portSpeedCableLength = dutQosConfig["portSpeedCableLength"]
```